index.html

```diff
@@ -2263,43 +2313,51 @@ <h5><code>queryaction</code></h5>
               </code>
             </li>
           </ul>
+	  </span>
           <pre class="example">
           GET /things/lamp/actions/fade/123e4567-e89b-12d3-a456-426655
           Host: mythingserver.com
           Accept: application/json
           </pre>
-          <p class="rfc2119-assertion"
+	  <span class="rfc2119-assertion"
             id="http-baseline-profile-protocol-binding-queryaction-5">
+	  <p>
             If a Web Thing receives an HTTP request following the format
             above and the Consumer has permission to query the corresponding
             <code>ActionStatus</code> resource, then upon successfully reading 
             the status of the action request it MUST send an HTTP response 
             with:
           </p>
-          <ul class="rfc2119-assertion"
-            id="http-baseline-profile-protocol-binding-queryaction-6">
+          <ul>
             <li>Status code set to <code>200</code></li>
             <li><code>Content-Type</code> header set to <code>application/json
               </code></li>
             <li>A body containing an <code>ActionStatus</code> object
               representing the current status of the action request, serialized 
             in JSON</li>
           </ul>
+	  </span>
           <pre class="example">
             HTTP/1.1 200 OK
             Content-Type: application/json
             {
               "status": "running"
             }
           </pre>
-          <p class="rfc2119-assertion"
-            id="http-baseline-profile-protocol-binding-queryaction-7">
-            If the queried action failed to execute, then the 
-            <code>status</code> member of the <code>ActionStatus</code> object 
-            MUST be set to <code>"failed"</code> and the <code>error</code>
-            member may optionally provide additional error information 
+	  <p>
+	  <span class="rfc2119-assertion"
+            id="http-baseline-profile-protocol-binding-queryaction-7a">
+            If the queried action failed to execute, then  
+            the <code>status</code> member of the <code>ActionStatus</code> object 
+            MUST be set to <code>"failed"</code>. 
+	  </span>
+	  <span class="rfc2119-assertion"
+            id="http-baseline-profile-protocol-binding-queryaction-7b">
+            If the queried action failed to execute, then  
+	    the <code>error</code>
+            member MAY provide additional error information 
```

Comment:
I think this harms readability, but I guess it's necessary for automation.

Reply:
well, the problem is not so much automation but when an assertion is taken out of context, i.e. in the implementation report, it's much better to have complete, self-contained sentences.

Reply:
anyway, assume this is just a comment so I'll mark as resolved.